### ISSUE TITLE
Update the customer tooltip component

### ISSUE THREAD

**Issue by vic-topcoder** [appirio-tech/connect-app#1445]
Right now we show some additional data that doesn't exist. Until that point let's remove the extra data from SFDC (it is hardcoded in the component).

Remove:
``` html
<div className="sf-data-bottom-container">
                    <div className="segment-data">
                      <span>Wipro Digital Jaipur / Topgear</span>
                    </div>
                    <div className="segment-data">
                      <span>0141-2211258</span>
                    </div>
                    <div className="segment-data">
                      <a href="https://google.com" target="_blank" rel="noopener noreferrer" className="ext-link">SDFC Lead Page</a>
                    </div>
````

Also, ProjectGridView should not hardcode the tooltip, but reuse a stand-alone component. At the moment it is like this:
![screen shot 10](https://user-images.githubusercontent.com/8484061/33971733-ecaedc44-e02e-11e7-8ba4-5c1a6d09bd2b.png)

Please use the tooltip components we have and simplify the components. Ideally this rendering component should be your container, and you'd reuse other components - tooltip, avatars, etc.

**Comment by ThomasKranitsas** [comment]
Is there anything else pending on this from my side?

**Comment by vic-topcoder** [comment]
We have to update the tooltips to use the stand-alone component instead of generating the content on the grid itself, I guess. But that'll be another task.
